op-program/Makefile

```diff
@@ -40,6 +47,8 @@ reproducible-prestate:
 	@cat ./bin/prestate-proof.json | jq -r .pre
 	@echo "MT-Cannon Absolute prestate hash: "
 	@cat ./bin/prestate-proof-mt.json | jq -r .pre
+	@echo "Cannon64 Absolute prestate hash: "
+	@cat ./bin/prestate-proof-mt64.json | jq -r .pre
```

Comment:
We should hold off on doing this. The 64-bit prestate will be changed by https://github.com/ethereum-optimism/optimism/pull/12569 and once it does it'll break the reproducible-prestate job.

Reply:
Good to merge this now that #12569 is merged.